exp/services/recoverysigner/cmd/keyset.go

```diff
@@ -0,0 +1,111 @@
+package cmd
+
+import (
+	"go/types"
+	"strings"
+
+	"github.com/google/tink/go/hybrid"
+	"github.com/google/tink/go/insecurecleartextkeyset"
+	"github.com/google/tink/go/integration/awskms"
+	"github.com/google/tink/go/keyset"
+	"github.com/spf13/cobra"
+	"github.com/stellar/go/support/config"
+	supportlog "github.com/stellar/go/support/log"
+)
+
+type KeysetCommand struct {
+	Logger         *supportlog.Entry
+	RemoteKEKURI   string
+	OutputFilePath string
```

Comment:
It's a bit confusing that these dependencies are a mix of externally provided dependencies by the thing that instantiates this class and dependencies collected via cobra's commands. I did the same thing as this in the db command and regretted it at the time. I don't feel strongly if you want to do this, but for the serve command I ended up building a set of options to pass to the command and that worked pretty well. 
https://github.com/stellar/go/blob/abd5c5f/exp/services/recoverysigner/cmd/serve.go#L97

Reply:
You're right. I've noticed the differences between the db command and the serve command. My take is that if there are only a few opts, go with the db stye. If there are a lot of ops, go with the serve style. I'm fine with either options here.

Note that I missed removing the `OutputFilePath` in my last commit.